kubernetes/spack/cdash-spack-io/db/pv.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: mysql
+spec:
+  storageClassName: gp2
```

Comment:
👍 We can't leave this field out altogether because our cluster is configured to use a default storage class (us-east-1a) which does dynamic provisioning. But setting it to the empty string seems to work.